Lesson_3/Program.cs

```diff
@@ -0,0 +1,268 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Lesson_3
+{
+    class Program
+    {
+        private static double RiseToThePower_1(double num, int degree)
+        {
+            double result = 1;
+            for (int i = 1; i <= degree; i++)
+            {
+                result *= num;
+            }
+
+            return Math.Round(result, 2);
+        }
+
+        private static int[] RoundDigits_2(int devider)
+        {
+            int[] result = new int[1000];
+            int pointer = 0;
+            for (int i = 1; i < 1000; i++)
+            {
+                if (i % devider == 0)
+                {
+                    result[pointer] = i;
+                    ++pointer;
+                }
+            }
+
+            return result;
+        }
+
+        private static int TotalCountOfPositiveDigits_3(double maxNum)
+        {
+            int result = 0;
+            for (int i = 1; i <= (int)Math.Sqrt(maxNum); i++)
```

Comment:
I think you don't need Sqrt here

Reply:
The biggest square integer from N is less than square root of number f.e. sqrt (9.5) = 3.08; pow(3, 2) = 9

Reply:
I know that you can use it for result search.
Sqrt is really slow operation. You can use bruteforce here (with thinking about conditions before codding)